src/components/common/FloatingVariablesSuggestions/service.ts

```diff
@@ -37,5 +38,14 @@ export const getScopedVariables = (appId, envId, clusterId) => {
 
     query += '}'
 
-    return get(`${Routes.SCOPED_GLOBAL_VARIABLES}${query}`)
+    const { result } = await get(`${Routes.SCOPED_GLOBAL_VARIABLES}${query}`) as ResponseType<ScopedVariableType[]>
+    if (!result) {
+        return []
+    }
+
+    if (hideObjectVariables) {
+        return result.filter((variable) => !variable.variableValue || typeof variable.variableValue.value !== 'object')
```

Comment:
pls verify for variableValue.value null case

Reply:
Not required as per api contract it will never be null